llvm/lib/Transforms/Vectorize/VPlan.cpp

```diff
@@ -1381,7 +1381,7 @@ void VPSlotTracker::assignNames(const VPlan &Plan) {
   if (Plan.VFxUF.getNumUsers() > 0)
     assignName(&Plan.VFxUF);
   assignName(&Plan.VectorTripCount);
-  if (Plan.BackedgeTakenCount)
```

Comment:
Unrelated change?

Reply:
not related to crash we saw in downstream compiler, but related to printing and assigning numbers to VPValues as few of our tests started to fail

Reply:
I see. As they are addressing separate issues, would be good to split it up, with the first one being NFC. Would be good to also have a test case for the numbering case